src/pages/privacy.tsx

```diff
@@ -0,0 +1,23 @@
+import { ComponentWrapperPage } from '@/components/near-org/ComponentWrapperPage';
+import { useBosComponents } from '@/hooks/useBosComponents';
+import { useDefaultLayout } from '@/hooks/useLayout';
+import { privacyDomainName } from '@/utils/config';
+import type { NextPageWithLayout } from '@/utils/types';
+
+const PrivacyPage: NextPageWithLayout = () => {
+  const components = useBosComponents();
+
+  return (
+    <ComponentWrapperPage
+      src={components.nearOrg.privacyPage}
+      meta={{ title: 'Privacy', description: '' }}
```

Comment:
@charleslavon what should we add to the `description` here? Are we good with `title`?